wiki/Medals/ko.md

```diff
@@ -1,24 +1,57 @@
+# 메달
+
+이전에 *업적*이라고 불리었던 메달은 유저가 특별한 목표를 달성했음을 나타내는 배지입니다. 현재 이 배지는, 종류별로 7개의 그룹으로 나뉘어져 있습니다: [Beatmap Packs](#beatmap-packs), [Skill](#skill), [Dedication](#dedication), [Mod Introduction](#mod-introduction), [Hush-Hush](#hush-hush), Beatmap Spotlights, Seasonal Spotlights.
+
+## Beatmap Packs
+
+**비트맵 팩(Beatmap Packs)** 메달은 각각의 [테마 비트맵 팩](https://osu.ppy.sh/beatmaps/packs?type=theme) 내의 **모든 비트맵을 어떤 난이도로든 적어도 한 번 이상 클리어**하면 얻을 수 있습니다. [비트맵 팩 목록](https://osu.ppy.sh/beatmaps/packs?type=theme) 내의 맵이 **`클리어한 맵`** 상태가 되고 배지를 얻으려면 반드시 [osu!standard](/wiki/osu!standard) 모드에서 플레이해야 합니다.
+
+### 요구 사항
+
+- 플레이어는 [테마 비트맵 팩](https://osu.ppy.sh/beatmaps/packs?type=theme)에서 다운로드한 맵 팩 내의 모든 비트맵을 어떤 난이도로든 적어도 한 번 이상 클리어해야 합니다.
+- 게임 모드는 osu!standard 이어야 합니다.
+- 테마 비트맵 팩을 다운로드하고, `.zip` 파일의 압축을 푼 뒤, 게임 설치 폴더 내의 `Songs` 폴더에 넣으세요.
+  - 여기서 osu!를 실행해 새롭게 추가된 비트맵을 인식할 수 있도록 해주세요.
+  - 다른 방법으로, 플레이어가 맵 팩 내의 곡을 수동으로 다운로드한 뒤 플레이할 수도 있습니다.
+- 바리에이션 곡(다른 제작자가 만든 맵)은 해당 곡을 플레이한 것으로 인정되지 않습니다.
+  **반드시** 맵 팩 내에 수록된 것과 곡, 제작자, 비트맵이 일치해야 합니다.
+- 해당 플레이어의 계정이 osu!클라이언트에 로그인되어 있어야 합니다 (웹사이트는 제외).
+
+### Notes
+
+- 모드는 Unranked 모드를 제외하고 자유롭게 사용할 수 있습니다. (예시) [Auto Pilot](/wiki/AP)).
+- 맵을 플레이한 뒤, [비트맵 팩 목록](https://osu.ppy.sh/beatmaps/packs?type=theme)에서 **`클리어한 맵`** 상태가 되었다면 로컬에서 해당 맵을 삭제해도 상관없습니다.
+
+## Skill
+
+**스킬(Skill)** 메달은 현재 세 그룹으로 나뉩니다: 콤보(Combo), 하이 랭커(High Ranker), 난이도 기반(Star-based).
+
+콤보(Combo) 메달은 osu!standard에서 특정 [콤보](/wiki/combo)를 도달하는 것으로 해금할 수 있으며, 어떤 노래든 달성이 가능합니다. 다만, [osu!taiko](/wiki/osu!taiko), [osu!catch](/wiki/osu!catch) 또는 [osu!mania](/wiki/osu!mania)에서는 메달을 해금할 수 없습니다.
+
+1000, 2000 콤보 메달을 해금하려면 Approved 상태의 비트맵을 찾아보는 게 좋습니다. **Ranked 또는 Approved 상태가 아닌 비트맵은 인정되지 않으니 주의하세요.**
+
 ---
-outdated: true
+
+하이 랭커(High-Ranker) 메달은 **게임 모드와 관계없이** 글로벌 랭킹에서 특정 순위에 도달하면 해금할 수 있습니다.
+
 ---
-# 어치브먼트
 
-**어치브먼트(Achievements) 또는 성과** 는 특정한 조건을 만족하였을때 받을수 있는 작은 뱃지들을 뜻한다.
+난이도 기반(Star-based) 메달은 특정 Star 이상의 비트맵을 클리어하며 해금할 수 있으며, *Pass*, *S* 두 개의 종류가 있습니다. *Pass* 메달은 비트맵을 클리어(EZ/NF/HT 제외)하면 얻을 수 있고, *S* 메달은 Pass 메달의 조건 + [풀 콤보](/wiki/FC)이면서 랭크를 S로 클리어하면 해금할 수 있습니다.
```

Comment:
Original content:
`The Star-based achivements are split into two groups, *Pass* and *S*. The *Pass* medals can be obtained by passing any map of a certain star difficulty. The *S* medals will be obtained if you pass the map with a S ranking whilst also having an [FC](/wiki/FC).`


Content in Korean (The grammar could be wrong):
`The Star-based achivements are split into two groups, *Pass* and *S*. The *Pass* medals can be obtained by passing any map of a certain star difficulty (excluding EZ/NF/HT). The *S* medals will be obtained if you meet the conditions of the Pass medal with a S ranking whilst also having an [FC](/wiki/FC).`